page-agreement.php

```diff
@@ -11,20 +11,13 @@
 <p>The agreement allows the company to offer customers ownCloud under another license, in exchange for money.
 This thus helps pay the bills for ownCloud contributors.</p>
 <h3>Does this mean that everybody who wants to contribute to ownCloud has to sign this agreement?</h3>
-<p>No. Only the <a href="https://github.com/owncloud/core">ownCloud core repository</a> is covered by this contributor agreement. ownCloud is build out of a lot of small modules
-called apps. We don't require a contributor agreement for the apps, only for the core, the desktop clients and android.<p>
-<h3>But I want to contribute to ownCloud and can&#39;t or don&#39;t want to sign a contributor agreement. What can I do?</h3>
+<p>No. 
```

Comment:
In the two lines below it neither looks right. including this line, there are 3 opening \<p\> tags, and none of them are closed. are they working dynamically? I'm not thats sure about my html knowledge, maybe it works this way.